mod/wet4/views/default/page/elements/head.php

```diff
@@ -95,149 +80,139 @@
 
 ?>
 
-	<!--[if lt IE 9]>
-		<script src="<?php echo $html5shiv_url; ?>"></script>
-	<![endif]-->
+<!--[if lt IE 9]>
+	<script src="<?php echo $html5shiv_url; ?>"></script>
+<![endif]-->
+
+<!--[if gt IE 8]>
+	<link rel="stylesheet" href="<?php echo $ie_url; ?>" />
+<![endif]-->
+
+<script><?php echo $elgg_init; ?></script>
 
 <?php
 
 foreach ($css as $url) {
-	echo elgg_format_element('link', array('rel' => 'stylesheet', 'href' => $url));
+  echo elgg_format_element('link', array('rel' => 'stylesheet', 'href' => $url));
 }
 
-?>
-	<!--[if gt IE 8]>
-		<link rel="stylesheet" href="<?php echo $ie_url; ?>" />
-	<![endif]-->
-
-	<script><?php echo $elgg_init; ?></script>
-<?php
 foreach ($js as $url) {
-
-    if (strpos($url,'jquery-1.11.0.min.js') !== false) {
-        //$url = 'https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js';
-    }
-
-    if (strpos($url,'require-2.1.10.min.js') !== false) {
-        //$url= str_replace($url,'require-2.1.10.min.js','require-2.1.20.min.js');
-    }
-
 	echo elgg_format_element('script', array('src' => $url));
 }
 
 echo elgg_view_deprecated('page/elements/shortcut_icon', array(), "Use the 'head', 'page' plugin hook.", 1.9);
-
 echo elgg_view_deprecated('metatags', array(), "Use the 'head', 'page' plugin hook.", 1.8);
 
 
-
-
-/*---------------------Web Experience Toolkit 4---------------------*/
+/*--------------------- Web Experience Toolkit 4 ---------------------*/
 
 ?>
 
-<!--<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">-->
-    <link rel="stylesheet" href="<?php echo $site_url ?>mod/wet4/views/default/css/awesome/font-awesome.min.css" type="text/css" />
-    <meta charset="utf-8" />
-		<!-- Web Experience Toolkit (WET) / Boîte à outils de l'expérience Web (BOEW)
+<!-- <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css"> -->
+<link rel="stylesheet" href="<?php echo $site_url ?>mod/wet4/views/default/css/awesome/font-awesome.min.css" type="text/css" />
+<meta charset="utf-8" />
```

Comment:
This might be a good guide https://github.com/wet-boew/wet-boew/blob/master/site/layouts/core.hbs
The metadata include is here https://github.com/wet-boew/wet-boew/blob/master/site/includes/metadata.hbs
The `asdf` include is here https://github.com/wet-boew/wet-boew/blob/master/site/includes/headresources.hbs#L5
EX: All metas, then style, then scripts.

PS: the manual HTML5 shiv isn't nessessary since the fallback WET IE js file includes it already.